src/vs/workbench/parts/update/electron-browser/update.ts

```diff
@@ -424,9 +424,9 @@ export class UpdateContribution implements IGlobalActivity {
 		let clazz: string | undefined;
 
 		if (state.type === StateType.AvailableForDownload || state.type === StateType.Downloaded || state.type === StateType.Ready) {
-			badge = new NumberBadge(1, () => nls.localize('updateIsReady', "New {0} update available.", product.nameShort));
+			badge = new NumberBadge(1, () => nls.localize('updateIsReady', "New {0} update available.", product.nameLong));
```

Comment:
please add `// {{SQL CARBON EDIT}}` tag when updating vs so we can track patches to VS Code source code.